.travis.yml

```diff
@@ -39,167 +33,13 @@ env:
-      jdk: openjdk8
-      if: type = cron
-    - os: linux
-      env: BUILD_ONLY_SUBMODULE=com.ibm.wala.ide.jsdt
-      jdk: openjdk8
-      if: type = cron
-    - os: linux
-      env: BUILD_ONLY_SUBMODULE=com.ibm.wala.ide.jsdt.tests
-      jdk: openjdk8
-      if: type = cron
-    - os: linux
-      env: BUILD_ONLY_SUBMODULE=com.ibm.wala.ide.tests
-      jdk: openjdk8
-      if: type = cron
-    - os: linux
-      env: BUILD_ONLY_SUBMODULE=com.ibm.wala.scandroid
-      jdk: openjdk8
-      if: type = cron
-    - os: linux
-      env: BUILD_ONLY_SUBMODULE=com.ibm.wala.shrike
-      jdk: openjdk8
-      if: type = cron
+      env: BUILD_SYSTEM=maven
     - os: linux
-      env: BUILD_ONLY_SUBMODULE=com.ibm.wala.util
-      jdk: openjdk8
-      if: type = cron
-    - os: osx
-      env: BUILD_ONLY_SUBMODULE=com.ibm.wala.cast
-      if: type = cron
-    - os: osx
-      env: BUILD_ONLY_SUBMODULE=com.ibm.wala.cast:cast
-      if: type = cron
-    - os: osx
-      env: BUILD_ONLY_SUBMODULE=com.ibm.wala.cast.java
-      if: type = cron
-    - os: osx
-      env: BUILD_ONLY_SUBMODULE=com.ibm.wala.cast.java.ecj
-      if: type = cron
-    - os: osx
-      env: BUILD_ONLY_SUBMODULE=com.ibm.wala.cast.java.test.data
-      if: type = cron
-    - os: osx
-      env: BUILD_ONLY_SUBMODULE=com.ibm.wala.cast.js
-      if: type = cron
-    - os: osx
-      env: BUILD_ONLY_SUBMODULE=com.ibm.wala.cast.js.html.nu_validator
-      if: type = cron
-    - os: osx
-      env: BUILD_ONLY_SUBMODULE=com.ibm.wala.cast.js.nodejs
-      if: type = cron
-    - os: osx
-      env: BUILD_ONLY_SUBMODULE=com.ibm.wala.cast.js.rhino
-      if: type = cron
-    - os: osx
-      env: BUILD_ONLY_SUBMODULE=com.ibm.wala.cast:smoke_main
-      if: type = cron
-    - os: osx
-      env: BUILD_ONLY_SUBMODULE=com.ibm.wala.cast:xlator_test
-      if: type = cron
-    - os: osx
-      env: BUILD_ONLY_SUBMODULE=com.ibm.wala.core
-      if: type = cron
-    - os: osx
-      env: BUILD_ONLY_SUBMODULE=com.ibm.wala.dalvik
-      if: type = cron
-    - os: osx
-      env: BUILD_ONLY_SUBMODULE=com.ibm.wala.ide
-      if: type = cron
-    - os: osx
-      env: BUILD_ONLY_SUBMODULE=com.ibm.wala.ide.jdt
-      if: type = cron
-    - os: osx
-      env: BUILD_ONLY_SUBMODULE=com.ibm.wala.ide.jdt.test
-      if: type = cron
-    - os: osx
-      env: BUILD_ONLY_SUBMODULE=com.ibm.wala.ide.jsdt
-      if: type = cron
-    - os: osx
-      env: BUILD_ONLY_SUBMODULE=com.ibm.wala.ide.jsdt.tests
-      if: type = cron
-    - os: osx
-      env: BUILD_ONLY_SUBMODULE=com.ibm.wala.ide.tests
-      if: type = cron
-    - os: osx
-      env: BUILD_ONLY_SUBMODULE=com.ibm.wala.scandroid
-      if: type = cron
+      jdk: openjdk11
     - os: osx
-      env: BUILD_ONLY_SUBMODULE=com.ibm.wala.shrike
-      if: type = cron
+      osx_image: xcode9.3  # Java 1.8
     - os: osx
-      env: BUILD_ONLY_SUBMODULE=com.ibm.wala.util
-      if: type = cron
+      osx_image: xcode9.4  # Java 13
```

Comment:
So is there no way to get JDK 11 on a Travis image?  It's good that this passes, but I don't know that we want to support anything WALA related on JDK 13 right now

Reply:
If this is the best we can do we can just stick with it

Reply:
Something changed on Travis CI’s side that now makes JDK 11 unavailable in any macOS image. [This Travis CI job suite](https://travis-ci.org/liblit/WALA/builds/659932950) tried macOS with each of the [17 available macOS base images](https://docs.travis-ci.com/user/reference/osx/#macos-version) (`xcode{6.4,7.3,8{,.3},9{,.{1,2,3,4}},10{,.{1,2,3}},11{,.{1,2,3}}}`) and with explicit `jdk:` requests for either `openjdk11` or `oraclejdk11`. That’s 34 possible job configurations. Travis CI silently ignored exactly half of them. Of the 17 that ran, `xcode9.3` and earlier always ended up with Java 1.8, and `xcode9.4` and later always ended up with Java 13.

Reply:
Others are [posting about this same problem](https://travis-ci.community/t/build-randomly-using-jdk-13-despite-osx-image-xcode10-1/6085/3).